apps/server/src/shared/repo/account/account.repo.integration.spec.ts

```diff
@@ -53,6 +53,20 @@ describe('account repo', () => {
 		});
 	});
 
+	describe('findByUsernameAndSystemId', () => {
+		it('findByUsernameAndSystemId, should return account', async () => {
```

Comment:
no need to repeat the methods name. That's what the surrounding `describe` is for.